testdata/react_with_loop/scripts/build.js

```diff
@@ -0,0 +1,193 @@
+'use strict';
+
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.BABEL_ENV = 'production';
```

Comment:
These scripts just come with the react app right? Did you use `create-react-app` or something else?

Reply:
I copied the app we had on the /react/ folder, and just removed the stepIn method and added the loop method instead. Every other file should be the same as far as I'm aware.

Reply:
Ok, it's no problem.